docs/en/tutorials/how_to_use_docker.md

```diff
@@ -21,6 +21,17 @@ cd mmdeploy
 docker build docker/GPU/ -t mmdeploy:0.1.0 --build-arg  VERSION=0.1.0
 ```
 
+For installing libs with the aliyun source, we can append `--build-arg USE_SRC_INSIDE=${USE_SRC_INSIDE}` to build command. 
+```
+# GPU for example
+cd mmdeploy
+docker build docker/GPU/ -t mmdeploy:0.1.0 --build-arg  USE_SRC_INSIDE=aliyun
```

Comment:
How about we use `USE_SRC_INSIDE=true` instead of `USE_SRC_INSIDE=aliyun`? And treat `USE_SRC_INSIDE` as a boolean value inside the Dockerfile.

Reply:
good idea